roxie/ccd/ccdserver.hpp

```diff
@@ -102,6 +102,8 @@ interface IFinalRoxieInput : extends IInputBase
     virtual bool gatherConjunctions(ISteppedConjunctionCollector & collector) { return false; }
     virtual unsigned numConcreteOutputs() const { return 1; }
     virtual IFinalRoxieInput * queryConcreteInput(unsigned idx) { assertex(idx==0); return this; }
+    virtual IEngineRowStream *queryConcreteOutputStream(unsigned idx) = 0;
```

Comment:
These parameters need renaming.  
idx should be whichInput.  
Really the prototype should be
virtual IEngineRowStream *queryConcreteOutputStream(unsigned whichInput, unsigned whichStrem) = 0;